app/assets/javascripts/admin/project/transfer_all_tasks_modal.js

```diff
@@ -0,0 +1,195 @@
+  transferActiveTasksOfProject,
+} from "admin/admin_rest_api";
+import type { APIUserType, APIProjectType, APIActiveUserType } from "admin/api_flow_types";
+import Toast from "libs/toast";
+import messages from "messages";
+
+const { Option } = Select;
+
+type Props = {
+  project: ?APIProjectType,
+  onCancel: Function,
+  visible: boolean,
+};
+
+type TableEntryType = {
+  key: string,
+} & APIActiveUserType;
+
+type State = {
+  isLoading: boolean,
+  users: Array<APIUserType>,
+  selectedUser: ?APIUserType,
+  usersWithActiveTasks: Array<APIActiveUserType>,
+};
+
+class TransferAllTasksModal extends React.PureComponent<Props, State> {
+  state = {
+    isLoading: false,
+    users: [],
+    selectedUser: null,
+    usersWithActiveTasks: [],
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  async fetchData() {
+    this.setState({ isLoading: true });
+    const users = await getUsers();
+    const activeUsers = users.filter(u => u.isActive);
+    let usersWithActiveTasks: Array<APIActiveUserType> = [];
+    if (this.props.project) {
+      usersWithActiveTasks = await getUsersWithActiveTasks(this.props.project.name);
+    }
+    this.setState({ isLoading: false, usersWithActiveTasks });
+    const sortedUsers = _.sortBy(activeUsers, "lastName");
+    this.setState({
+      users: sortedUsers,
+    });
+  }
+
+  async transferAllActiveTasks() {
+    if (this.state.selectedUser && this.props.project) {
+      try {
+        await transferActiveTasksOfProject(this.props.project.name, this.state.selectedUser.id);
+        if (this.state.selectedUser) {
+          Toast.success(
+            `${messages["project.successful_active_tasks_transfer"]} ${
+              this.state.selectedUser.lastName
+            }, ${this.state.selectedUser.firstName}`,
+          );
+        }
+      } catch (e) {
+        Toast.error(messages["project.unsuccessful_active_tasks_transfer"]);
+      }
+      this.props.onCancel();
+    }
+  }
+
+  renderTableContent() {
+    const activeUsersWithKey: Array<TableEntryType> = this.state.usersWithActiveTasks.map(
+      activeUser => {
+        const userWithKey = {
+          email: activeUser.email,
+          activeTasks: activeUser.activeTasks,
+          key: activeUser.email,
+        };
+        return userWithKey;
+      },
+    );
+    const columns = [
+      {
+        title: "user's email",
+        dataIndex: "email",
+        key: "email",
+      },
+      {
+        title: "number of active tasks",
+        dataIndex: "activeTasks",
+        key: "activeTasks",
+      },
+    ];
+    return (
+      <Table
+        columns={columns}
+        dataSource={activeUsersWithKey}
+        rowKey="email"
+        pagination={false}
+        size="small"
+      />
+    );
+  }
+
+  handleSelectChange = (userId: string) => {
+    const selectedUser = this.state.users.find(user => user.id === userId);
+    this.setState({ selectedUser });
+  };
+
+  renderFormContent() {
+    return (
+      <Select
```

Comment:
You copied that code from the transfer_task_modal, right? If you feel like it, you can share this code by creating a new "<UserSelectionComponent />" component. The approach would be very similar to the `TeamSelectionComponent` which abstracts the same thing for teams. If you think it's to much for this PR, please add a comment above this line here (e.g., "Todo: create a <UserSelectionComponent /> component which is used here and in transfer_task_modal").